src/reducer.test.js

```diff
@@ -125,4 +132,45 @@ describe('reducer', () => {
       });
     });
   });
+
+  describe('changeLoginField', () => {
+    const initialState = {
+      loginField: {
+        email: '',
+        password: '',
+      },
+    };
+
+    context('changes email field', () => {
+      const state = reducer(
+        initialState,
+        changeLoginField({ name: 'email', value: 'test@test.co.kr' }),
+      );
+
+      expect(state.loginField.email).toBe('test@test.co.kr');
+      expect(state.loginField.password).toBe('');
+    });
+
+    context('changes password field', () => {
+      const state = reducer(
+        initialState,
+        changeLoginField({ name: 'password', value: 'test' }),
+      );
+
+      expect(state.loginField.email).toBe('');
+      expect(state.loginField.password).toBe('test');
+    });
+  });
+
+  describe('setAccessToken', () => {
+    it('changes the accessToken', () => {
+      const initialState = {
+        accessToken: '',
+      };
+
+      const state = reducer(initialState, setAccessToken('SAMPLE_TOKEN'));
```

Comment:
```suggestion
      const state = reducer(initialState, setAccessToken('ACCESS_TOKEN'));
```

저희 팀은 보통 이렇게 의미를 더 드러내요. 이 부분엔 'ACCESS_TOKEN이 들어간다'는 의미죠.

Reply:
가독성 면에서 훨씬 보기 좋네요!!